lib/ansible/modules/cloud/ovirt/ovirt_storage_domains.py

```diff
@@ -426,7 +428,9 @@ def main():
             choices=['present', 'absent', 'maintenance', 'unattached'],
             default='present',
         ),
+        id=dict(default=None),
         name=dict(required=True),
+        import_=dict(type='bool', default=False),
```

Comment:
We use an `imported` state for importing a template, maybe we should use it also here. Also I think that other then `iscsi` and `fcp` storage can be imported,no?